app/components/site_search_form_component.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class SiteSearchFormComponent < ViewComponent::Base
+  def presenter
+    Blacklight::SearchBarPresenter.new(controller, SearchAcrossController.blacklight_config)
+  end
+
+  def search_fields
```

Comment:
Got this functionality from https://github.com/projectblacklight/blacklight/blob/d566efebb088ef87bffeb5a6b5a12c5017adda5f/app/components/blacklight/search_bar_component.rb#L53-L57

Reply:
Should this class extend from `Blacklight::SearchBarComponent` and inherit this code?

Reply:
I don't think so. Would be a lot of code we aren't using.

Reply:
@jcoyne I didn't realize that the Blacklight::SearchBarPresenter was being deprecated. I have update the component to extend Blacklight::SearchBarComponent. Thanks for pointing me in the right direction.